src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -164,6 +164,20 @@ public void drive(Translation2d translation, Rotation2d rotation, boolean fieldO
 		}
 	}
 
+	public void driveWithDoubleSupplierTurn(
```

Comment:
What's the point in this method? Nothing seems to call it, and it behaves identically to the `drive()` method above it (except the bug fix, which should be handled in a separate PR).

Reply:
Probably tested it and it didn't work, will remove

Reply:
Removed